_includes/step-indicator.html

```diff
@@ -37,14 +40,12 @@
     {% endfor %}
   </ol>
   <div class="usa-step-indicator__header">
-    <h4 class="usa-step-indicator__heading">
-      <span class="usa-step-indicator__heading-counter"
-        ><span class="usa-sr-only">Step</span>
-        <span class="usa-step-indicator__current-step">{{ current_step }}</span>
-        <span class="usa-step-indicator__total-steps">of {{ steps.size }}</span> </span
-      ><span class="usa-step-indicator__heading-text"
-        >{{ steps[currentstepindex].text }}</span
-      >
-    </h4>
+<h4 class="usa-step-indicator__heading">
+<span class="usa-step-indicator__heading-counter">
+<span class="usa-sr-only">Step</span>
+<span class="usa-step-indicator__current-step">
+  {% if substep %}{{ substep }}{% else %}{{ current_step }}{% endif %}</span> {% if hide_steps ==false %}<span class="usa-step-indicator__total-steps">of {{ steps.size }}</span> </span>{% endif %}
```

Comment:
```suggestion
  {% if substep %}{{ substep }}{% else %}{{ current_step }}{% endif %}</span> {% if hide_steps ==false %}<span class="usa-step-indicator__total-steps">of {{ steps.size }}</span>{% endif %}</span>
```

I ***think*** this last `</span>` needs to be closed outside of the `{% if ...` because it closes the `span` opened on line 44?